### PR TITLE
[Agent] add acl_gids to pcap

### PR DESCRIPTION
**Pull request by TomatoMr**
### This PR is for:

- Agent

#### Checklist
- [ ] Added unit test.
#### Backport to branches
- main

### CODE REVIEW

File: agent/src/handler/mod.rs [hunk 1]
```diff
@@ -139,6 +144,7 @@ impl PacketHandler {
                     packet: packet.packet[..max_raw_len].to_vec(),
                     flow_id: packet.flow_id,
                     timestamp: Duration::from_nanos(packet.timestamp),
+                    acl_gids: acl_gids.list().iter().cloned().collect(),
```
Comment (rvql) [inline agent/src/handler/mod.rs:147]: `acl_gids.0.cloned()`

Comment (TomatoMr) [reply agent/src/handler/mod.rs:147]: acl_gids.0 会报错：field `0` of `U16Set` is private

Comment (rvql) [reply agent/src/handler/mod.rs:147]: 哦，那你用list，但不需要iter再collect

Comment (rvql) [reply agent/src/handler/mod.rs:147]: https://doc.rust-lang.org/std/vec/struct.Vec.html#impl-From%3C%26%5BT%5D%3E-for-Vec%3CT%2C%20Global%3E

Comment (TomatoMr) [reply agent/src/handler/mod.rs]: ok，已修改

**Review by rvql — APPROVED** [review]

**Pull request merged**